### DISCUSSION TITLE
Why are there two copies of the course in the repo?

### DISCUSSION THREAD

**Discussion by Glinte** [mrdbourke/pytorch-deep-learning#494] [General]
one copy in root and another in /docs

**Comment by mrdbourke** [comment]
Hi @Glinte,

Good question! 

Due to the course being built with Jupyter Notebooks, a conversion step happens when commits happen to turn the Jupyter Notebooks into docs (see here: https://www.learnpytorch.io/).

This step happens in the GitHub Action here: https://github.com/mrdbourke/pytorch-deep-learning/blob/main/.github/workflows/make_docs.yml 

The docs get built from the `docs/` folder.

As of now it seems inefficient but it's one of the only simple ways I've found to make the course + documentation with Jupyter Notebooks simultaneously without huge overheads.

If you know a better way, I'd love to hear about it :)
